lib/cocoapods/installer/xcode/multi_pods_project_generator.rb

```diff
@@ -26,6 +26,8 @@ def generate!
 
           integrate_targets(target_installation_results.pod_target_installation_results)
           wire_target_dependencies(target_installation_results)
+
+          container_project ||= Pod::Project.new(sandbox.project_path)
```

Comment:
It may better to check `nil` on `Installer.`

https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/installer.rb#L297

However, it is difficult to test this behavior. So `MultiPodsProjectGenerator` tries to open existing projects.